src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -84,21 +84,21 @@ Map<UUID, Object> ids = (Map<UUID, Object>) request.getAttribute("ids");
               Instant instant = u.getCreationTime();
               Date myDate = Date.from(instant);
         %>
-          <li><strong><%= myDate %>:</strong> <%= u.getName() + " joined! " %></li>
+          <li><strong><%= ActivityFeedServlet.DateFormatter.formatDateToString(myDate,"EEE MMM dd yyyy hh:mm:ss a z","PST") %>:</strong> <%= u.getName() + " joined! " %></li>
```

Comment:
if EEE MMM dd yyyy.... is the only date format you're going to use, it may be good to hardcode this into the servlet

OR use https://docs.oracle.com/javase/8/docs/api/java/time/format/FormatStyle.html the formatstyle enum which you can use to get a date formatter (https://docs.oracle.com/javase/8/docs/api/java/time/format/class-use/FormatStyle.html) rather than hardcoding a date format

Reply:
Done, I hardcoded it